lib/mumukit/platform/organization/profile.rb

```diff
@@ -19,4 +23,16 @@ def locale_json
   def logo_url
     @logo_url ||= 'https://mumuki.io/logo-alt-large.png'
   end
+
+  def banner_url
+    @banner_url ||= logo_url
```

Comment:
I'd use `||` instead of `||=`. Otherwise, behavior can get tricky:

```ruby
org.banner_url = nil
org.logo_url = 'logo.png'

org.banner_url 
# returns 'logo.png`, which is ok

org.logo_url = 'logo2.png'
# returns 'logo.png`, which is not ok, since it should be logo2.png
```